src/main/java/core/mvc/tobe/HandlerMappingComposite.java

```diff
@@ -13,16 +12,15 @@
 
     private Set<HandlerMapping> handlerMappings = new LinkedHashSet<>();
 
-    public HandlerMappingComposite(HandlerMapping ... handlerMappings) {
+    public HandlerMappingComposite(HandlerMapping... handlerMappings) {
         this.handlerMappings.addAll(Arrays.asList(handlerMappings));
     }
 
     @Override
-    public Object getHandler(HttpServletRequest request) {
+    public HandlerExecution getHandler(HttpServletRequest request) {
         return handlerMappings.stream()
                 .filter(handlerMapping -> handlerMapping.getHandler(request) != null)
-                .map(handlerMapping ->  handlerMapping.getHandler(request))
-                .findAny()
-                .get();
+                .map(handlerMapping -> handlerMapping.getHandler(request))
+                .findAny().orElseThrow(() -> new PageNotFoundException("404 Page Not Found RequestURI = " + request.getRequestURI(), request.getRequestURI()));
```

Comment:
예외 처리와 친절한 예외 메시지 좋네요. 👍 
stream 작성시에는 라인 별로 구분을 해주면 IntelliJ 에서 타입을 가이드 해주기 때문에 코드를 파악하기 더 용이하다고 생각해요 😄 
```suggestion
                .findAny()
                .orElseThrow(() -> new PageNotFoundException("404 Page Not Found RequestURI = " + request.getRequestURI(), request.getRequestURI()));
```